spec/index.html

```diff
@@ -9701,16 +9701,17 @@ <h5>Sum</h5>
               be 6.0 (float).</p>
             <div class="defn">
               <p><b>Definition: <span id="defn_aggSum">Sum</span></b></p>
-              <pre class="code nohighlight">numeric Sum(sequence S)</pre>
-              <p>L = Flatten(S)</p>
-              <p>Sum(S) = Sum(L)</p>
-              <p>Sum(L) = op:numeric-add(L<sub>1</sub>, Sum(L<sub>2..n</sub>)) if <a href="#defn_Card">Card</a>(L) &gt;
+              <pre class="code nohighlight">numeric <var>Sum</var>(sequence <var>S</var>)</pre>
+              <p>|Sum|(|S|) = <var>Sum'</var>(|L|),
```

Comment:
What about writing: `Sum(S) = Sum'(Flatten(S))`  where Sum'(L) is defined recursively as follows:` instead? I would find it slightly clearer.

Reply:
> What about writing: Sum(S) = Sum'(Flatten(S)) where Sum'(L) is defined recursively as follows:` instead? I would find it slightly clearer.

Not really, because then it would not be clear in the rest of the definition what L is.